virttest/utils_test/qemu.py

```diff
@@ -1774,9 +1774,49 @@ def check_memory(self, vm=None):
         self.os_type = self.params.get("os_type")
         guest_mem_size = super(MemoryHotplugTest, self).get_guest_total_mem(vm)
         vm_mem_size = self.get_vm_mem(vm)
-        threshold = vm_mem_size * 0.06
+        default_threshold = vm_mem_size * 0.06
+        threshold = float(self.params.get("threshold", default_threshold))
         if abs(guest_mem_size - vm_mem_size) > threshold:
```

Comment:
And if I'm right, please configure 'threshold' param to increase or decrease obligate address. thanks.